hack/tests/subcommand-scorecard.sh

```diff
@@ -16,10 +16,13 @@ set -ex
 pushd test/test-framework
 
 # test to see if scorecard fails when version is v1alpha2 and when external plugins are configured
-operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep '^.*error validating plugin config.*$'
+!(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep '^.*error validating plugin config.*$')
```

Comment:
Why ! was added?

Reply:
To make things easier to debug when this fails, it may be more clear to do something like:
```bash
if ! $(operator-sdk scorecard --version v1alpha2 --config "$CONFIG_PATH" |& grep -q '^.*error validating plugin config.*$'); then
    echo "expected scorecard to fail when version is v1alpha2 and when external plugins are configured"
    exit 1
fi